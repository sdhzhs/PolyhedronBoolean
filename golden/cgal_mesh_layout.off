OFF
16 18 0

-5.000000000000000e+00 -5.000000000000000e+00 0.000000000000000e+00
-5.000000000000000e+00 -5.000000000000000e+00 3.000000000000000e-01
5.000000000000000e+00 -5.000000000000000e+00 3.000000000000000e-01
5.000000000000000e+00 -5.000000000000000e+00 0.000000000000000e+00
5.000000000000000e+00 5.000000000000000e+00 0.000000000000000e+00
-5.000000000000000e+00 5.000000000000000e+00 0.000000000000000e+00
0.000000000000000e+00 1.000000000000000e+00 0.000000000000000e+00
1.000000000000000e+00 1.000000000000000e+00 0.000000000000000e+00
1.000000000000000e+00 0.000000000000000e+00 0.000000000000000e+00
0.000000000000000e+00 0.000000000000000e+00 0.000000000000000e+00
-5.000000000000000e+00 5.000000000000000e+00 3.000000000000000e-01
5.000000000000000e+00 5.000000000000000e+00 3.000000000000000e-01
1.154700538379252e-01 1.154700538379252e-01 2.000000000000000e-01
1.154700538379252e-01 8.845299461620748e-01 2.000000000000000e-01
8.845299461620748e-01 8.845299461620748e-01 2.000000000000000e-01
8.845299461620748e-01 1.154700538379252e-01 2.000000000000000e-01
4  1 0 3 2
3  5 9 0
3  6 9 5
3  5 4 6
3  4 7 6
3  4 8 7
3  4 3 8
3  8 3 9
3  9 3 0
4  5 0 1 10
4  10 1 2 11
4  2 3 4 11
4  4 5 10 11
4  6 13 12 9
4  7 14 13 6
4  14 7 8 15
4  12 15 8 9
4  13 14 15 12

