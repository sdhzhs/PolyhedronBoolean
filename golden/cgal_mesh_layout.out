8, 6
0.000000000000000e+00 1.000000000000000e+00 0.000000000000000e+00
1.000000000000000e+00 1.000000000000000e+00 0.000000000000000e+00
1.000000000000000e+00 0.000000000000000e+00 0.000000000000000e+00
0.000000000000000e+00 0.000000000000000e+00 0.000000000000000e+00
1.154700538379252e-01 8.845299461620748e-01 2.000000000000000e-01
1.154700538379252e-01 1.154700538379252e-01 2.000000000000000e-01
8.845299461620748e-01 1.154700538379252e-01 2.000000000000000e-01
8.845299461620748e-01 8.845299461620748e-01 2.000000000000000e-01
0 1 2 3 
4 5 6 7 
0 4 7 1 
1 7 6 2 
2 6 5 3 
3 5 4 0 
